### PR TITLE
Add limit on comment text when showing on main page

### PR DESCRIPTION
**Pull request by FaheemAhmedNoor**

### CODE REVIEW

File: app/assets/javascripts/comment.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+$(function(){
```
Comment (ShehrozIrfan) [inline app/assets/javascripts/comment.js:1]: Hey @FaheemAhmedNoor ,
The `JavaScript` solution is good. But I'm wondering is there any `Ruby` method available to achieve the same thing or we need to do it in `JavaScript` and no `Ruby` way?

File: app/views/comments/index.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
                 <% @comments.each do |comment| %>
                   <tr>
                     <td>
-                      <%= link_to comment_path(comment.id), class: 'text-dark' do %> <%= comment.text %><% end %>
+                      <%= link_to comment_path(comment.id), class: 'text-dark link' do %> <%= comment.text %><% end %>
```
Comment (ShehrozIrfan) [inline app/views/comments/index.html.erb:12]: Please use a definitive class name, like `link` is very generalized.
You can use `comment-text-with-limit`

**Comment by FaheemAhmedNoor** [comment]
I've search for the ruby solution but I was not able to find it.

**Comment by ShehrozIrfan** [comment]
Hey @FaheemAhmedNoor ,
We can use ` comment.text.truncate(255)` and that's it. No need to do a lot of stuff.

**Comment by ShehrozIrfan** [comment]
@FaheemAhmedNoor 
Is it possible for you to add the above updates on this weekend?

**Comment by ShehrozIrfan** [comment]
#62

**Pull request closed without merging**